calicoctl/resourcemgr/resourcemgr.go

```diff
@@ -21,14 +21,15 @@ import (
 	"reflect"
 	"strings"
 
-	"io/ioutil"
+	"io"
```

Comment:
Copyright statements in all your updated files should be updated to include 2017.